src/app.ts

```diff
@@ -406,6 +409,7 @@ export function newApp({
         SPID_LOG_QUEUE_NAME
       );
       const spidLogCallback = makeSpidLogCallback(spidQueueClient);
+      hrStart = process.hrtime();
```

Comment:
Should we use `process.hrtime.bigint()`? It seems to have better precision without using a tuple